views/partials/component/type/demos.html

```diff
@@ -28,6 +28,7 @@ <h3 class="o-registry-ui__component--demo-title">{{capitalise title}}</h3>
 				{{#if display.html}}
 					<div class="o-tabs__tabpanel" id="{{title}}-html-{{@index}}">
 						{{#if display.plainHTML}}
+							<button class="select-html">SELECT FULL CODE SNIPPET</button>
```

Comment:
`text-transform: uppercase` can achieve the visual effect without changing the meaning -- i.e. some screen-readers would read this letter-by-letter.